heralding/reporting/file_logger.py

```diff
@@ -38,6 +39,12 @@ def __init__(self, session_logfile, auth_logfile):
 
         logger.info('File logger started, using files: %s and %s', auth_logfile, session_logfile)
 
+        # SSH protocol's filehandler ,writer for auxiliary data
+        ssh_aux_field_names = AuxiliaryData.get_filelog_fields('ssh')
```

Comment:
the logger should have no hardcoded knowledge in regards to which protocol it is logging auxiliary date on behalf of.

I think this needs to be handled dynamically somehow.

Reply:
Can i use dynamic variable creation ?